tadasets/shapes.py

```diff
@@ -180,5 +181,11 @@ def infty_sign(n=100, noise=None, seed=None):
 
     if noise:
         X += noise * np.random.randn(n, 2)
-    
+    if angle is not None:
+        try:
+            assert angle >= -np.pi and angle <= 2*np.pi
+        except AssertionError:
+            raise RotationAngleNotInRangeError(angle, 0, "pi")
```

Comment:
I think the custom error hear is little overkill. Would we lose anything by having a single line?

```
assert angle >= -np.pi and angle <= 2*np.pi, "Angle {angle} not in range. Angle should be in the range {min_angle} <= angle <= {max_angle}".format(...)
```

Reply:
No, great point! I will fix that.